Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rustposix"
 version = "0.1.0"
-authors = ["Nicholas Smith Renner <nrenner@nyu.edu>"]
+authors = ["Nicholas Smith Renner <nrenner@nyu.edu>", "Jonathan Eli Singer <jes965@nyu.edu>"]
 edition = "2018"
```

Comment:
Should the edition be changed as well? Also, it would be fun if I could have my name on here as well! I totally understand if not, though, as I am not a leader :).

Reply:
Yeah @TristanB22 you can add your name on your next PR.